config/cloud-config/node-config-template.yaml

```diff
@@ -73,19 +73,28 @@ write_files:
     permissions: 0755
     content: |
       #! /usr/bin/bash
-      
+      CALICO_VERSION=$(/usr/bin/etcdctl get /calico-install/version) >/dev/null 2>&1 || CALICO_VERSION="0.12.0"
```

Comment:
nope normaly thats done via env variables and it usualy runs even on a worker just in proxy mode so localhost is tryed first .. just the way etcd works :)